Editor/MobiledgeXEditorWindow.cs

```diff
@@ -349,8 +384,9 @@ public async Task<bool> CheckCredentials()
             {
                 // Register and find cloudlet:
                 clog("Registering to DME ...", "");
-                checkResult = await integration.Register();
-                bool foundCloudlet = await integration.FindCloudlet();
+                checkResult = await integration.Register(Region == "" ? "" : Region, Region == "" ? (uint) 0 : DME_PORT);
```

Comment:
dmeHost can just be Region. Don't think you need the ternary operator

Reply:
As discussed it if Nearest is chosen it will perform A regular RegisterAndFindCloudlet("",0)

Reply:
This comment is about the first parameter. It can just be integration.Register(Region, Region == "" ? uint(0) : DME_PORT);

Reply:
This is confusing to the eye, ternary ops should make things easier to understand at first glance, not harder.

How about you just put it can another line like at 273, and then use that?